app/src/main/res/layout/clock_user_bug.xml

```diff
@@ -1,39 +1,48 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent" android:layout_height="match_parent">
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
 
     <TextClock
+        android:id="@+id/clock"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:id="@+id/clock"
         android:layout_alignParentTop="true"
         android:layout_centerHorizontal="true"
+        android:fontFamily="sans-serif-light"
         android:format12Hour="h:mm"
         android:format24Hour="k:mm"
+        android:shadowColor="@color/black"
+        android:shadowRadius="4"
+        android:textColor="@color/white"
         android:textSize="40sp"
-        android:fontFamily="sans-serif-light" />
+        tools:text="13:37" />
 
     <LinearLayout
```

Comment:
The linter warning on this line basically says: use a <TextView> with `drawableStart`. Not sure how loading with glide works in that case though.

Reply:
It seems like this is possible, but seems much more complex than the current approach for little to no gain?

Reply:
Fair enough, it shouldn't matter AFAIK